code/modules/reagents/Chemistry-Recipes/Others.dm

```diff
@@ -75,6 +75,33 @@
 	required_reagents = list("capsaicin" = 1, "ethanol" = 5)
 	result_amount = 5
 
+/datum/chemical_reaction/soapification
+	name = "Soapification"
+	id = "soapification"
+	result = null
+	required_reagents = list("liquidgibs" = 10, "lye"  = 10) // requires two scooped gib tiles
+	required_temp = 374
+	result_amount = 1
+	mob_react = 1
+
+/datum/chemical_reaction/soapification/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	new /obj/item/weapon/soap/homemade(location)
+	return
+
+/datum/chemical_reaction/candlefication
+	name = "candlefication"
+	id = "candlefication"
+	result = null
+	required_reagents = list("liquidgibs" = 5, "stabilizing_agent"  = 5) //
```

Comment:
ehhh, I'd suggest against using stabilizing agent in a recipe like this, it's only really supposed to be used for stabilizing violent reactions.